### PR TITLE
409: Admin creator wizard

### PR DESCRIPTION
**Pull request by maxb94**

### CODE REVIEW

File: src/settings.ts [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+    name: "A Theme Camp",
+    description: ["Clickable map", "Embed zooms, venues, and so on"],
+  },
+  {
+    name: "Performance Venue",
+    description: ["Performance space", "Tables with video chat"],
+  },
+  {
+    name: "Art Piece",
+    description: ["Clickable map", "Embed zooms, venues, and so on"],
```
Comment (antoineeudes) [inline src/settings.ts:39]: not exactly the Art Piece venue but don't worry I'll change it

File: src/api/admin.ts [hunk 2]
```diff
@@ -13,20 +13,31 @@ export interface EventInput {
   price: number;
 }
 
-export interface VenueInput {
+interface Question {
```
Comment (mathieud) [inline src/api/admin.ts:16]: Can we use the existing question type?

Comment (mathieud) [reply src/api/admin.ts:16]: Actually name is purely internal, don't think it should be part of the form (can slugify the text again)

File: src/settings.ts (hunk 1 above)
Comment (maxb94) [reply src/settings.ts:39]: This copy comes from Ed's designs

File: src/pages/Account/Venue/AdvancedDetailsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import { AdvancedVenueInput } from "api/admin";
+
+interface AdvancedDetailsProps {
+  register: ReturnType<typeof useForm>["register"];
+  control: ReturnType<typeof useForm>["control"];
+  values: Partial<AdvancedVenueInput>;
+  errors: FieldErrors<Partial<AdvancedVenueInput>>;
+}
+
+export const AdvancedDetailsForm: React.FC<AdvancedDetailsProps> = (props) => {
```
Comment (mathieud) [inline src/pages/Account/Venue/AdvancedDetailsForm.tsx:12]: Should this be a template specific form (or will be?)

Comment (maxb94) [reply src/pages/Account/Venue/AdvancedDetailsForm.tsx:12]: not for the moment, but maybe in the future. If so, the template would need to be passed as a prop

**Pull request merged**